.github/workflows/push.yaml

```diff
@@ -46,14 +49,20 @@ jobs:
           go-version: ${{ env.GO_VERSION }}
           cache: false
 
+      - name: Compute base image
+        id: img
+        run: |
+          IMG="quay.io/${{ env.QUAY_NAMESPACE }}/${{ matrix.image_base }}"
+          echo "base=${IMG}" >> ${GITHUB_OUTPUT}
+
```

Comment:
The stuff under the `build-workspaces` job doesn't touch anything related to the kubesaw toolchain.  I'm trying to understand what these changes are supposed to accomplish.  What do they add that the old version of this job didn't do?

I'm fine with this change, just puzzled about its purpose.

Reply:
They makes the life way easier when testing the changes to ci in a fork

Reply:
We can move them to another PR if we prefer, I forgot them here